common/models/service-instance.json

```diff
@@ -53,7 +53,6 @@
     },
     "cpus": {
       "type": "string",
-      "default": "cpus",
```

Comment:
Because this default gets persisted in the DB as the intended size... overriding any more dynamic notion of what the default size should be. Like `STRONGLOOP_CLUSTER`. Now, if you set a per-instance cpus value, it will have it, if you don't, it will be undefined, and pm decides what size to use.